app/views/home/index.html.haml

```diff
@@ -23,7 +23,7 @@
     %h3 Step 3
     %ul
       %li
-        Read the #{ link_to 'rules', 'http://mitrecyberacademy.org/terms-of-service'}. If you don't follow them, you could be #{ link_to 'disqualified', 'http://i.imgur.com/VXSxAAE.gifv'}.
+        Read the #{ link_to 'rules', 'http://mitrecyberacademy.org/competitions/rules/index.html'}. If you don't follow them, you could be #{ link_to 'disqualified', 'http://i.imgur.com/VXSxAAE.gifv'}.
```

Comment:
Need to link to both the rules and the TOS, should read "Read the rules and the Terms of Service"